example-clients/jack_control

```diff
@@ -393,8 +392,11 @@ def main():
                 result = control_iface.RemoveSlaveDriver(name)
             else:
                 print("Unknown command '%s'" % arg)
-        except dbus.DBusException as e:
-            print("DBus exception: %s" % str(e))
+    except dbus.DBusException as e:
+        print("DBus exception: %s" % str(e))
+        print("Please check the dbus logs, most likely in the systemd journal (view with journalctl)")
```

Comment:
jackdbus does not output to systemd journal by default, so how is this "most likely" ?

Reply:
jack install itself as a dbus activatable service

```
# process org.jackaudio.service.in -> org.jackaudio.service   
obj = bld(                                                    
        features = 'subst',                                   
        source = 'org.jackaudio.service.in',                  
        target = 'org.jackaudio.service',                     
        install_path = '${DBUS_SERVICES_DIR}/',               
        BINDIR = bld.env['PREFIX'] + '/bin')                  
```

and it looks like the dbus logs go to systemd journal on most distros, by default, so when the activation fails, the stderr for the jackdbus application _probably_ ends up in the journal. I'm not sure there's a way to do any better than that; as far as I can tell, dbus doesn't really have a well defined place to redirect stderr for dbus-activated services